react-validated-input-component/my-app/src/ValidationForm.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+export default function ValidationForm() {
+  const [password, setPassword] = useState('');
+  function handleChange(e) {
+    setPassword(e.target.value);
+  }
+  let val = password.length >= 8 ? 'is-valid' : 'is-invalid';
+  let txt = 'A password is required';
+  if (password.length < 8 && password.length !== 0) {
+    txt = 'Your password is too short';
+  }
+  if (password.length >= 8) {
+    txt = '';
+  }
+
+  return (
+    <div>
+      <form className="form-group position relation needs-validation">
```

Comment:
For this exercise, you are writing a reusable component that another component can use in a form. You don't want the `form` tag here.